### ISSUE TITLE
shaper play corrupts new line terminators

### ISSUE THREAD

**Issue by wenditer** [arno49/shaper#18]
template:
```
        localconfig:
          atg:
            Initial.properties:
              initialServices+: '{{ aux.config.atg.Initial.initialServices_plus }}'
```

playbook:
```
    aux
      config:
        atg:
          Initial:
            initialServices_plus: '\
              /atg/tracking/UsageTrackingService,\
              /atg/sitemap/google/GoogleSitemapGeneratorService,\
              /atg/sitemap/google/GoogleSitemapWriterService'
```
after running `shaper play`  result looks like:
```
        localconfig:
          atg:
            Initial.properties:
              initialServices+: \ /atg/tracking/UsageTrackingService,\atg/sitemap/google/GoogleSitemapGeneratorService,\/atg/sitemap/google/GoogleSitemapWriterService
```

Expected result should look exactly the same as in variable with all new line terminators in place

**Comment by arno49** [comment]
Fixed in 0.0.8

**Comment by wenditer** [comment]
still reproduced
